resqpy/surface/_surface.py

```diff
@@ -694,15 +685,20 @@ def set_from_point_set(self,
             t = triangulate.dt(p_xy_e[:, :2], container_size_factor = convexity_parameter * 1.1)
         log.debug('number of triangles: ' + str(len(t)))
         if make_clockwise:
```

Comment:
Let's make make_clockwise True in a unit test, so the make_all_clockwise_xy() gets to run at least. It shouldn't change any behaviour of the code afterwards.